src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -327,14 +360,20 @@ class __attribute__((visibility("default"))) TvgLottieAnimation
 
 private:
     string                 errorMsg;
-    unique_ptr<SwCanvas>   canvas = nullptr;
+    unique_ptr<Canvas>   canvas = nullptr;
     unique_ptr<Animation>  animation = nullptr;
     string                 data;
     uint8_t*               buffer = nullptr;
     uint32_t               width = 0;
     uint32_t               height = 0;
     float                  psize[2];         //picture size
     bool                   updated = false;
+
+    #ifdef THORVG_WG_RASTER_SUPPORT
```

Comment:
```
#ifdef THORVG_SW_RASTER_SUPPORT
    uint8_t*               buffer = nullptr;
    uint32_t               width = 0;
    uint32_t               height = 0;
#endif
#ifdef THORVG_WG_RASTER_SUPPORT
    WGPUInstance instance{};
    WGPUSurface surface{};
#endif

Reply:
Done. however width, height are still needed for resize function to check whether size changed